apps/emqx_bridge/src/emqx_bridge.erl

```diff
@@ -89,7 +89,8 @@
     T == pulsar_producer;
     T == oracle;
     T == iotdb;
-    T == kinesis_producer
+    T == kinesis_producer;
```

Comment:
For easy maintenance, it is customary to keep one application only exposing one bridge module and named it with the backend name, like `mysql` `pgsql` and so on.
Or split it into several applications, just like the `pgsql` `timescale` and `matrix`, although they use the same driver, but are different applications.

Reply:
`kinesis_producer` is the existing bridge,  this PR only adds `emqx_bridge_greptimedb`.